workflows/mag.nf

```diff
@@ -112,6 +107,8 @@ workflow MAG {
 
     if (!params.keep_phix) {
         ch_phix_db_file = Channel.value(file("${params.phix_reference}"))
+    } else {
```

Comment:
Is this the fix for the [phix bug](https://github.com/nf-core/mag/issues/740)? If so would appreciate to split this out into a separate PR: